src/main/java/frc/robot/RobotContainer.java

```diff
@@ -199,16 +201,22 @@ private void configureButtonBindings() {
             () -> driveSubsystem.drive(-leftJoystick.getY() * 0.7, rightJoystick.getX() * 0.7),
             driveSubsystem));
 
-        btnLauncherSolenoid.whenPressed(
-            new AutomaticShootCommand(TARGET_VELOCITY, NUM_BALLS_LOADED, shooterSubsystem)
+        btnLauncherSolenoid.whenHeld(
+            new AutomaticShootCommand(TARGET_VELOCITY, NUM_BALLS_LOADED, shooterSubsystem),
+            true
         );
 
         btnIntakeSolenoid.whenPressed(new InstantCommand(intakeSubsystem::extend, intakeSubsystem))
         .whenInactive(new InstantCommand(intakeSubsystem::retract, intakeSubsystem));
       
-        btnIntakeOut.whenHeld(new InstantCommand(() -> intakeSubsystem.spin(0.5, 0.7), intakeSubsystem))
+        btnIntakeOut.whenHeld(new InstantCommand(() -> intakeSubsystem.spin(6.5, 4.5), intakeSubsystem))
             .whenInactive(new InstantCommand(() -> intakeSubsystem.spin(0, 0), intakeSubsystem), true);
-        btnIntakeIn.whenHeld(new InstantCommand(() -> intakeSubsystem.spin(-0.5, -0.7), intakeSubsystem))
+        btnIntakeIn.whenHeld(new InstantCommand(() -> intakeSubsystem.spin(-6.5, -4.5), intakeSubsystem))
```

Comment:
You guys should discuss with the mechanics team to see if you can use gearing to reduce the speed so that the motors are running at full power

Reply:
That's actually a very good idea. We also had issues with balls getting jammed too easily, so that would fix that a bit.

Reply:
Same with the driving. I think I saw a `.7` multiplier in Bobby's PR. Increasing gearing will help push other robots around

Reply:
We're going to try changing the gearing sometime soon. For now I'm going to merge this PR.